src/computeFontSize.ts

```diff
@@ -0,0 +1,63 @@
+import * as vscode from "vscode";
+
+export interface FontSize {
+  fontSize: number;
+  fontWidth: number;
+  fontHeight: number;
+}
+
+export default async function (): Promise<FontSize> {
+  const fontRatios = await getFontRatios();
+  const fontSize = getFontSize();
+  return {
+    fontSize,
+    fontWidth: fontSize * fontRatios[0],
+    fontHeight: fontSize * fontRatios[1],
+  };
+}
+
+function getFontRatios(): Promise<[number, number]> {
```

Comment:
I'd make a type rather than returning a tuple.  That way you can name them to make it clearer what's being returned, eg `fontWidthToFontSizeRatio`

Reply:
Yeah I know. I was just lazy.